ceph-releases/jewel/centos/7.4.1708/daemon-base/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# CEPH BASE IMAGE
+# CEPH VERSION: Jewel
+# CEPH VERSION DETAIL: 10.2.x
+
+FROM BASEIMAGE
+MAINTAINER Sébastien Han "seb@redhat.com"
+
+ENV CEPH_VERSION jewel
+
+# Install Ceph
+ARG PACKAGES="unzip ceph-mon ceph-osd ceph-mds ceph-base ceph-common ceph-radosgw rbd-mirror device-mapper sharutils etcd kubernetes-client e2fsprogs wget"
```

Comment:
daemon base should not include install wget unzip etc.

Reply:
ack